scripts/gee

```diff
@@ -2381,6 +2381,49 @@ function gee__diff() {
   fi
 }
 
+##########################################################################
+# grep command
+##########################################################################
+
+_register_help "grep" "Greps the current branch." <<'EOT'
+Usage: gee grep [options] <expression>
+
+Searches the current branch for the specified expression.
+
+If the ripgrep tool is installed, then this command invokes:
+
+    rg "$@" "${BRANCH_ROOT_DIR}"
+
+If ripgrep is not installed, this command falls back on the grep utility,
+invoked to be similar in operation to ripgrep, like this:
+
+    grep -r --exclude-dir=.git "$@" "${BRANCH_ROOT_DIR}"
+
+If the `gee bash_setup` environment is loaded, `grg` is an alias for this
+command.
+
+Example of use:
+
+    grg -t bazel unit_test
```

Comment:
This example searches for `bazel` inside the `unit_test` directory? Will it work if `$BRANCH_ROOT_DIR` is not empty below?

Reply:
"-t bazel" is an option for ripgrep - it tells ripgrep to only search files of type "bazel" (ie. *.bzl WORKSPACE BUILD BUILD.bazel etc).  It's searching for the expression "unit_test".

Maybe not the best example.